server/semantics-service/src/main/java/edu/illinois/ncsa/incore/service/semantics/model/Column.java

```diff
@@ -6,7 +6,8 @@
 @Embedded
 public class Column {
 
-    private String name;
+    @Property("name")
+    private static String name;
```

Comment:
any reason why you need to use the property annotaion? isn't it already "name"?

Reply:
Thanks for pointing out. It's removed.